docs/Makefile

```diff
@@ -46,3 +46,34 @@ markdownlint:
 
 markdownlint-fix:
 	$(MDL_CMD) --entrypoint="markdownlint-cli2-fix" davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} "**/*.md" 
+
+# Define the directories
+OPERATOR_DIR := operator
+CONFIG_DIR := $(OPERATOR_DIR)/config
+HELM_TEMPLATES_DIR := helm/chart/templates
+METRICS_OPERATOR_DIR := metrics-operator
+
+.PHONY: resources
+
+# New target to automate the steps for both operators
+resources: generate-manifests copy-manifests update-crd-docs
+
+generate-manifests:
+	@echo "Generating manifests..."
+	cd $(OPERATOR_DIR) && make generate && make manifests
+	cd $(METRICS_OPERATOR_DIR) && make generate && make manifests
+
+copy-manifests:
+	@echo "Copying manifests to Helm..."
+	cp $(CONFIG_DIR)/crd/bases/*.yaml $(HELM_TEMPLATES_DIR)/
```

Comment:
you will need to use kustomize to generate the crds here. just copying the yaml files will not be enough since there are some patch files that are applied to the CRD yamls by kustomize